### PR TITLE
rust 1.48.0

### PR DESCRIPTION
**Pull request by its-pointless**
test build don't merge

### CODE REVIEW

**Comment by buttaface** [comment]
We've cleared up another 13 GB of disk space on the CI, please rebase and run this through CI again. You shouldn't run out of disk space anymore.

**Comment by buttaface** [comment]
Rebased to get it to use the new space on the CI.

**Comment by buttaface** [comment]
Alright, it builds fine. Just let me know when and we can merge the LLVM pull, followed by these compilers that depend on it.

**Comment by its-pointless** [comment]
should be okay

**Comment by its-pointless** [comment]
not okay wasm32 isn't working

**Comment by buttaface** [comment]
That lld issue is on the CI or when using the rust compiler on an Android device?

**Comment by its-pointless** [comment]
on device

**Comment by buttaface** [comment]
Alright, I'll merge these three LLVM-related pulls and we'll find out.

**Comment by buttaface** [comment]
Since this links against the libllvm package, I will merge it last, once the new libllvm 11 package has been pushed to the mirrors by the CI.

**Comment by buttaface** [comment]
LLVM 11 is up on the package mirrors: should I merge this now or do you want to wait and try the build artifacts after the latest commit to this pull is done building?

File: packages/rust/build.sh [hunk 1]
```diff
@@ -92,7 +96,9 @@ termux_step_make_install() {
 }
```
Comment (landfillbaby) [inline packages/rust/build.sh:96]: just above this (line 94) there's still a reference to libLLVM-10.0.1.so
i think we need an _LLVM_VER variable

Comment (buttaface) [reply packages/rust/build.sh:97]: Added it.

**Comment by buttaface** [comment]
Rebased and fixed the symbolic link issue, go ahead and merge if you don't need to see the CI result.

**Pull request merged**